tests/test_constant.py

```diff
@@ -0,0 +1,8 @@
+from egcg_core import constants as cst
+
+
+def test_constants_uniqueness():
+    constants = [item for item in dir(cst) if not item.startswith("__")]
+    uniq_constants = set()
+    for constant in constants:
+        assert constant not in uniq_constants
```

Comment:
This will just assert that `constant` is not in an empty set, which will always be True!